build.sbt

```diff
@@ -4,9 +4,12 @@ val spymemcached    = "net.spy" % "spymemcached" % "2.12.3"
 val h2databaseTest  = "com.h2database" % "h2" % "1.4.196" % Test
 
 lazy val baseSettings = Seq(
-  scalaVersion := "2.12.7",
-  crossScalaVersions := Seq("2.11.12", "2.12.7"),
-  parallelExecution in Test := false
+  scalaVersion := "2.12.8",
+  crossScalaVersions := Seq("2.11.12", "2.12.8", "2.13.0-M5"),
+  parallelExecution in Test := false,
+  // Workaround until omnidoc gets published for Scala 2.13
+  // http://central.maven.org/maven2/com/typesafe/play/play-omnidoc_2.13/
+  PlayKeys.playOmnidoc := false
```

Comment:
Could keep the regular behaviour of omnidoc for non-2.13 versions if anyone was bothered.

`PlayKeys.playOmnidoc := PlayKeys.playOmnidoc.value && !scalaBinaryVersion.value.startsWith("2.13")`

Reply:
Your example code doesn't work. I get errors. Please try yourself.

Reply:
Also I think it does not really matter for a module like play2-memcached if `playOmnidoc` is used or not. `playOmnidoc` does not make any use of it anyway and usually it only matters in the application itself (and even there usually no one makes us of it, despite maybe running the Play docs offline).

Reply:
You're quite right that it doesn't work here - I'd been testing it on a project with the Play plugin enabled, but it doesn't work here in `commonSettings` as there's no previous value of `playOmnidoc` to work from.

You're also right that it doesn't really matter for the reasons you describe. If it was an issue then you'd be able to use this in the projects that did use the plugin (I think it's just `javaSample` and `scalaSample` that are affected) but I think your PR is good to go as-is.

Reply:
:+1: